### PR TITLE
enhance: Seeding & Testing

### PR DESCRIPTION
**Pull request by andytaylorkainos**
This PR is to fix some duplication when seeding. Stop tests from running seeding per test and to make the console output easier to digest; the console ONLY displays entities created or updated

### CODE REVIEW

File: test_helpers/clients.py [hunk 1]
```diff
@@ -579,30 +606,21 @@ def create_standard_application_case(self, organisation: Organisation, reference
...
-            advice.denial_reasons.set(["1a", "1b", "1c"])
-
-        advice.save()
-        return advice
-
-    @staticmethod
-    def create_good_country_decision(case, goods_type, country, decision):
-        GoodCountryDecision(case=case, good=goods_type, country=country, decision=decision).save()
+    def create_end_user_advisory_case(self, note: str, reasoning: str, organisation: Organisation):
+        eua_query = self.create_end_user_advisory(note, reasoning, organisation)
```
Comment (BenAndrew-Kainos) [inline test_helpers/clients.py]: Make this oneline please

File: static/management/SeedCommand.py [hunk 2]
```diff
@@ -54,11 +62,19 @@ def update_or_create(model: models.Model, rows: list):
...
-            if obj.exists():
-                obj.update(**row)
-            else:
+            obj_id = row["id"]
+            obj = model.objects.filter(id=obj_id)
+            if not obj.exists():
                 model.objects.create(**row)
+                print(dict(row))
+            else:
+                # Can not delete the "id" key-value from `rows` as it will manipulate the data which is later used in
```
Comment (BenAndrew-Kainos) [inline static/management/SeedCommand.py]: Please move this code to a function called something like `check_obj_needs_updating`

File: static/management/SeedCommand.py [hunk 3]
```diff
@@ -54,11 +62,19 @@ def update_or_create(model: models.Model, rows: list):
...
         for row in rows:
-            obj = model.objects.filter(id=row["id"])
-            if obj.exists():
-                obj.update(**row)
-            else:
+            obj_id = row["id"]
+            obj = model.objects.filter(id=obj_id)
+            if not obj.exists():
                 model.objects.create(**row)
+                print(dict(row))
```
Comment (BenAndrew-Kainos) [inline static/management/SeedCommand.py]: Please add to the print to say `UPDATED` or `CREATED`

Comment (andytaylorkainos) [reply static/management/SeedCommand.py]: Done!

File: static/management/SeedCommand.py (hunk 2 above)
Comment (andytaylorkainos) [reply static/management/SeedCommand.py]: Done!

File: test_helpers/clients.py (hunk 1 above)
Comment (andytaylorkainos) [reply test_helpers/clients.py]: done!

File: static/management/commands/seedall.py [hunk 4]
```diff
@@ -31,8 +31,9 @@ class Command(SeedCommand):
     pipenv run ./manage.py seedall
     """
 
-    help = "Executes all seed operations"
-    success = "All seed operations executed!"
+    help = "executes all seed operations"
+    info = "EXECUTING ALL SEED OPERATIONS"
```
Comment (stephanieboyd) [inline static/management/commands/seedall.py]: Why we shouting this?

Comment (stephanieboyd) [reply static/management/commands/seedall.py]: Kay, shouting looks good.

File: static/management/commands/seedorgusers.py [hunk 5]
```diff
@@ -93,18 +93,14 @@ def _extract_names_from_email(exporter_user_email: str):
...
-        + user.first_name
-        + '", "last_name": "'
-        + user.last_name
-        + '", id": "'
-        + str(user.id)
-        + '"}'
+    user_org = UserOrganisationRelationship.objects.get_or_create(
+        user=user, organisation=organisation, status=UserStatuses.ACTIVE
     )
+    if user_org[1]:
```
Comment (stephanieboyd) [inline static/management/commands/seedorgusers.py]: Can we make it clear that this is a boolean outlining creation success?

Comment (andytaylorkainos) [reply static/management/commands/seedorgusers.py]: Done!

**Pull request merged**